packages/react-components/react-carousel-preview/stories/src/Carousel/CarouselAutoplayControlled.stories.tsx

```diff
@@ -78,7 +78,7 @@ export const AutoplayControlled = () => {
         >
           <CarouselButton navType="prev" />
           <CarouselAutoplayButton
-            autoplay={enableAutoplay}
+            checked={enableAutoplay}
             onClick={() => {
```

Comment:
Updated the vars to 'checked' here to match the underlying ToggleButton props but they don't provide an onCheckedChange for now, think we should keep this specific to autoplay callback and event data in case that changes.